tests/test_viz.py

```diff
@@ -351,9 +351,6 @@ def test_plot_distance_exceptions(ocx, api_data):
 @pytest.mark.parametrize(
     "metric,dissimilarity_metric,smacof",
     [
-        ("abundance_w_children", "weighted_unifrac", 0.7595),
```

Comment:
Why were these test cases removed?

Reply:
This was in error -- PR to add them back in : https://github.com/onecodex/onecodex/pull/412